include/zephyr/usb/class/usb_hub.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
@mateusz- Just noticed `* Copyright (c)` is missing, please add. I will take it to the host PR asap. Thanks.

Reply:
@mateusz- I can not take it over otherwise because we don not merge PR containing source code files without copyright, upstream PR would be blocked.